supabase/migrations/20240430155721_rename_profiles_plus_to_users_plus_and_include_profile_id.sql

```diff
@@ -0,0 +1,42 @@
+drop view if exists "public"."profiles_plus";
+
+create or replace view "public"."users_plus" as  SELECT users.id AS user_id,
```

Comment:
Renamed the users_plus view to be consistent with the other views which are named off of the primary table that they are built from